tooltool.py

```diff
@@ -919,49 +898,19 @@ def process_command(options, args):
         return False
 
 
-# fetching api:
-#   http://hostname/algorithm/hash
-#   example: http://people.mozilla.org/sha1/1234567890abcedf
-# This will make it possible to have the server allow clients to
-# use different algorithms than what was uploaded to the server
-
-# TODO: Implement the following features:
-#   -optimization: do small files first, justification is that they are faster
-#    and cause a faster failure if they are invalid
-#   -store permissions
-#   -local renames i.e. call the file one thing on the server and
-#    something different locally
-#   -deal with the cases:
-#     -local data matches file requested with different filename
-#     -two different files with same name, different hash
-#   -?only ever locally to digest as filename, symlink to real name
-#   -?maybe deal with files as a dir of the filename with all files in that dir
-#    as the versions of that file - e.g. ./python-2.6.7.dmg/0123456789abcdef and
-#    ./python-2.6.7.dmg/abcdef0123456789
-
-
-def main():
-    # Set up logging, for now just to the console
-    ch = logging.StreamHandler()
-    cf = logging.Formatter("%(levelname)s - %(message)s")
-    ch.setFormatter(cf)
-
+def main(argv, _skip_logging=False):
     # Set up option parsing
     parser = optparse.OptionParser()
```

Comment:
This reminds me that it would be great to add python2.6 to .travis.yml and test tooltool using python 2.6.